### ISSUE TITLE
Visualization option - to see all tracks at once

### ISSUE THREAD

**Issue by roian** [kristinbranson/APT#265]
At the moment you can only see the tracking results for one target animal at a time. It would be nice to have an option to see all of them at once, maybe with different symbols, to see errors.

**Comment by allenleetc** [comment]
Yeah think we have some infrastructure for this. With CPR (or labels?) Think there was a time you could shift-click and highlight multiple rows in the targets table and see labels/trks for multiple targets. Don't quote on me on it and almost def doesn't work anymore.

**Comment by allenleetc** [comment]
Note this now works for imported tracking but not regular tracking results.

**Comment by arobie** [comment]
We really need to get this working for regular tracking results!

**Comment by allenleetc** [comment]
Agreed I hit this as well and had to export/re-import. This should be a fairly quick enhancement.

**Comment by allenleetc** [comment]
@arobie This should be in let me know if it works. Enabled viz of predictions for all targets simultaneously. See View> menu to toggle showing for primary tgt only.

Did some testing but a lot of surface area here so not sure I got everything. Think I saw one minor thing when I first opened a project and tracked the preds didn't appear until I toggled "Hide Predictions" or something to that effect. But the issue went away.

Fuller tracking results comparison could still use some enhancements eg across trackers, multiple imports, differentiated cosmetics etc. Some of this infrastructure is in but some thought still required. Any usage scenarios appreciated!
